menus/darwin.cson

```diff
@@ -165,6 +164,7 @@
         submenu: [
           { label: 'Open In Dev Mode…', command: 'application:open-dev' }
           { label: 'Run Package Specs', command: 'window:run-package-specs' }
+          { label: 'Reload Window', command: 'window:reload' }
```

Comment:
Just an idea – I wonder which of these is the most commonly used command for package developers? If it's Reload Window, maybe it would be worth putting it first? Otherwise, maybe alphabetical order would be best.